config/src/config/execution_config.rs

```diff
@@ -83,7 +86,10 @@ mod test {
 
     #[test]
     fn test_some_and_load_genesis() {
-        let fake_genesis = Transaction::WriteSet(WriteSetMut::new(vec![]).freeze().unwrap());
+        let fake_genesis = Transaction::WriteSet(ChangeSet::new(
+            WriteSetMut::new(vec![]).freeze().unwrap(),
+            vec![],
+        ));
```

Comment:
Are we doing this anywhere else in the code? Could you check and if so, maybe we have a new "constructor" in WriteSet called empty?

Reply:
There were a bunch of locations that created a ChangeSet with a WriteSet and an empty event vector. Any suggestions for the name? Like `ChangeSet::empty_event`?